### PR TITLE
Expired Levy is now included in transaction lines returned for downlo…

### PR DESCRIPTION
**Pull request by tom-gough**
…ad file

### CODE REVIEW

File: src/SFA.DAS.EAS.Employer_Financial.Database/StoredProcedures/GetAllTransactionDetailsForAccountByDate.sql [hunk 1]
```diff
@@ -91,4 +90,32 @@ UNION ALL
...
+			CONVERT(INT, NULL)								AS ApprenticeTrainingCourseLevel,
+			CONVERT(DECIMAL(18,4), NULL)					AS PaidFromLevy,
+			CONVERT(DECIMAL(18,4), NULL)					AS EmployerContribution,
+			CONVERT(DECIMAL(18,4), NULL)					AS GovermentContribution,
+			tl.Amount										AS Total
+	FROM	[employer_financial].TransactionLine tl
+			LEFT JOIN [employer_financial].[TransactionLineTypes] tlt
+				ON tlt.TransactionType = 3
+	WHERE	tl.AccountId = @accountId 
+			AND tl.TransactionType IN (1, 2, 5)
```
Comment (elliotritchie) [inline src/SFA.DAS.EAS.Employer_Financial.Database/StoredProcedures/GetAllTransactionDetailsForAccountByDate.sql]: Is this clause needed?

File: src/SFA.DAS.EAS.Employer_Financial.Database/StoredProcedures/GetAllTransactionDetailsForAccountByDate.sql [hunk 2]
```diff
@@ -91,4 +90,32 @@ UNION ALL
...
+
+SELECT	tl.DateCreated									AS DateCreated,
+			tlt.[Description]								AS TransactionType,
+			tl.EmpRef										AS PayeScheme,
+			NULL											AS PayrollYear,
+			NULL											AS PayrollMonth,
+			tl.LevyDeclared									AS LevyDeclared,
+			tl.EnglishFraction								AS EnglishFraction,
+			tl.Amount - (LevyDeclared * EnglishFraction)	AS TenPercentTopUp,
+			CONVERT(VARCHAR(MAX), NULL)						AS TrainingProvider,
```
Comment (elliotritchie) [inline src/SFA.DAS.EAS.Employer_Financial.Database/StoredProcedures/GetAllTransactionDetailsForAccountByDate.sql]: Are the conversions needed or could `NULL` be returned instead?

File: src/SFA.DAS.EAS.Employer_Financial.Database/StoredProcedures/GetAllTransactionDetailsForAccountByDate.sql [hunk 3]
```diff
@@ -91,4 +90,32 @@ UNION ALL
...
+UNION ALL
+
+SELECT	tl.DateCreated									AS DateCreated,
+			tlt.[Description]								AS TransactionType,
+			tl.EmpRef										AS PayeScheme,
+			NULL											AS PayrollYear,
+			NULL											AS PayrollMonth,
+			tl.LevyDeclared									AS LevyDeclared,
+			tl.EnglishFraction								AS EnglishFraction,
+			tl.Amount - (LevyDeclared * EnglishFraction)	AS TenPercentTopUp,
```
Comment (elliotritchie) [inline src/SFA.DAS.EAS.Employer_Financial.Database/StoredProcedures/GetAllTransactionDetailsForAccountByDate.sql]: Should a value be returned here or could `NULL` be returned instead?

File: src/SFA.DAS.EAS.Employer_Financial.Database/StoredProcedures/GetAllTransactionDetailsForAccountByDate.sql [hunk 4]
```diff
@@ -91,4 +90,32 @@ UNION ALL
...
+
+UNION ALL
+
+SELECT	tl.DateCreated									AS DateCreated,
+			tlt.[Description]								AS TransactionType,
+			tl.EmpRef										AS PayeScheme,
+			NULL											AS PayrollYear,
+			NULL											AS PayrollMonth,
+			tl.LevyDeclared									AS LevyDeclared,
+			tl.EnglishFraction								AS EnglishFraction,
```
Comment (elliotritchie) [inline src/SFA.DAS.EAS.Employer_Financial.Database/StoredProcedures/GetAllTransactionDetailsForAccountByDate.sql]: Should a value be returned here or could `NULL` be returned instead?

File: src/SFA.DAS.EAS.Employer_Financial.Database/StoredProcedures/GetAllTransactionDetailsForAccountByDate.sql [hunk 5]
```diff
@@ -91,4 +90,32 @@ UNION ALL
...
+				meta.ApprenticeshipCourseLevel) AS funds
+
+UNION ALL
+
+SELECT	tl.DateCreated									AS DateCreated,
+			tlt.[Description]								AS TransactionType,
+			tl.EmpRef										AS PayeScheme,
+			NULL											AS PayrollYear,
+			NULL											AS PayrollMonth,
+			tl.LevyDeclared									AS LevyDeclared,
```
Comment (elliotritchie) [inline src/SFA.DAS.EAS.Employer_Financial.Database/StoredProcedures/GetAllTransactionDetailsForAccountByDate.sql]: Should a value be returned here or could `NULL` be returned instead?

**Comment by tom-gough** [comment]
superseded by https://github.com/SkillsFundingAgency/das-employerapprenticeshipsservice/pull/1599

**Pull request closed without merging**